### PR TITLE
+1 icon

### PR DESCRIPTION
**Pull request by Alex123098c**
# Description

### Added

PowerAudio Plus (`power.amp.musicplayer.pi.audioplayer`)

## Contributor's checklist
- [x] I followed [the Lawnicons guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

**Comment by thismoon** [comment]
It would be better to link it to `[generic_player.svg](https://github.com/LawnchairLauncher/lawnicons/blob/develop/svgs%2Fgeneric_player.svg)` instead of creating a new icon in this case

File: app/assets/appfilter.xml [hunk 1]
```diff
@@ -12233,4 +12233,5 @@
   <item component="ComponentInfo{com.starbucks.co/com.starbucks.co2.ui.intro.IntroActivity}" drawable="starbucks" name="스타벅스 ~~ Starbucks" />
   <item component="ComponentInfo{viva.republica.toss/viva.republica.toss.splash.SplashActivity}" drawable="toss" name="토스 ~~ Toss" />
   <item component="ComponentInfo{space.takashima.keypo/org.mozc.android.inputmethod.japanese.LauncherActivity}" drawable="keyboard" name="ﾀｸﾄ ｷｰﾎﾞｰﾄﾞ ~~ Takt keyboard" />
+  <item component="ComponentInfo{power.amp.musicplayer.pi.audioplayer/xsoftstudio.musicplayer.ActivityMain}" drawable="poweraudio_plus" name="PowerAudio Plus"/>
```
Comment (x9136) [inline app/assets/appfilter.xml:12236]: `"/>` → `" />`

Need a space here.

File: svgs/poweraudio_plus.svg
Comment (x9136) [inline svgs/poweraudio_plus.svg:1]: The easiest way is to remove the icon and make the link the way **thismoon** suggested.

If you want to keep the icon, then consider the following:
1. The icon size is incorrect.
2. The icon looks shifted to the left due to the emptiness on the right.
3. The stroke width of such minimalistic icons increases to 14px.

**Review by x9136 — CHANGES_REQUESTED** [review]

**Comment by x9136** [comment]
#2229

**Pull request closed without merging**